plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -777,15 +835,6 @@ public boolean downloadPlugin(Plugin plugin, File location) {
         }
         String pluginDownloadUrl = getPluginDownloadUrl(plugin);
         boolean successfulDownload = downloadToFile(pluginDownloadUrl, plugin, location);
-        if (!successfulDownload) {
```

Comment:
Does it still work with plugins like docker which is published as docker-plugin.hpi?

Reply:
Yes but you need to use the plugins name, docker-plugin

Reply:
I had thought there was a mismatch between the hpi and the artifactId for some plugins, but that is not the case. We just need to make sure and document this as a breaking change.